tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -547,7 +561,7 @@ def log_call_i(x):
         return log(x)
 
     f1 = epyccel(log_call_i, language = language)
-    x = randint(low=min_float, high=1e6)
+    x = randint(low=min_int, high=max_int)
```

Comment:
`log` is a function which is applied to positive numbers. It shouldn't be used for negative numbers. If it is used for negative numbers then it returns a `RuntimeWarning` and `nan`, but `nan`s cannot be compared:
```
In [2]: a=np.log(-5)
/home/emily/.local/bin/ipython3:1: RuntimeWarning: invalid value encountered in log
  #!/usr/bin/python3

In [3]: a
Out[3]: nan

In [4]: b=np.log(-4)
/home/emily/.local/bin/ipython3:1: RuntimeWarning: invalid value encountered in log
  #!/usr/bin/python3

In [5]: b
Out[5]: nan

In [6]: a==b
Out[6]: False

In [7]: a-b
Out[7]: nan

In [8]: np.isclose(a,b)
Out[8]: False
```

You should use `x = randint(low=0, high=max_int)`